src/protocol/mod.rs

```diff
@@ -9,7 +9,9 @@ pub(crate) use protocol::ascii::AsciiProtocol;
 pub(crate) use protocol::binary::BinaryProtocol;
 use std::collections::HashMap;
 use stream::Stream;
-use value::{FromMemcacheValue, ToMemcacheValue};
+use value::{FromMemcacheValueExt, ToMemcacheValue};
+
+pub type CasId = u64;
```

Comment:
Seems `CasId` is just a `u64` alias, I think we should keep it simple and let users just using the native `u64` type, and to keep a unified API style while the `set`'s expiration is just an `u32`.

Reply:
I agree with you that keeping a consistent API is important but I think `CasId` is more descriptive than `u64`. Users can still use `u64` without any extra boilerplate.

Reply:
I think we should keep it as a native `u64`, and maybe add a trait for this and expire param someday.